### ISSUE TITLE
Pale Moon/Firefox extension?

### ISSUE THREAD

**Issue by hook321** [kissarat/never-lose#3]
Is there a similar extension for Pale Moon or Firefox? If not, is there a chance that a Pale Moon/Firefox version of this will be made?

**Comment by kissarat** [comment]
No one interested in this project...

**Comment by hook321** [comment]
What if I find people that are interested in it?

**Comment by kissarat** [comment]
Good idea. I want to rewrite extension using Vue.js and external server
